src/controllers/screens.js

```diff
@@ -0,0 +1,16 @@
+const { Prisma } = require('@prisma/client')
+const prisma = require('../utils/prisma')
+
+const createScreen = async (req, res) => {
+  const body = req.body
+  console.log('body', body);
+   console.log(typeof body.number) 
```

Comment:
Nice way of troubleshooting and checking you're getting the datatypes you expect!

Reply:
Thank you:)